analysis/webservice/apidocs/openapi.yml

```diff
@@ -118,8 +118,24 @@ paths:
 
           required: true
           schema:
-            type: string
+            type: array
```

Comment:
Confirmed this does translate into comma separated values when multiple are selected.

Reply:
Oh wow, that is awesome! Can you pull this out the matchup endpoint, put it in a reference object, and use the reference obj for both the match and subset endpoints?

Reply:
Oh wow, that is awesome! Can you pull this out the matchup endpoint, put it in a reference object, and use the reference obj for both the match and subset endpoints?

Reply:
I can look into that, yes

Reply:
Done

Reply:
> > > > NexusHandlers can now support multiple paths.
> > > 
> > > 
> > > Just curious, what is the purpose of this?
> > 
> > 
> > I figured in the event that an endpoint is being renamed we can do it without breaking anything that uses the existing endpoint. It's not really being used in this pr anymore though.
> 
> Since we're no longer using it, let's remove it. But we if we need it in the future, we can reintroduce it since we'll have this PR to reference

Ok. It's been removed.